app/views/handlers/feedback.py

```diff
@@ -19,6 +19,10 @@ class FeedbackLimitReached(Exception):
     pass
 
 
+class FeedbackUploadFailedException(Exception):
```

Comment:
To be consistent with the other two exceptions.
```suggestion
class FeedbackUploadFailed(Exception):
```